<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModeControlWidgetClass</class>
 <widget class="QWidget" name="ModeControlWidgetClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>192</width>
    <height>40</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>40</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ModeControlWidget</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>1</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QPushButton" name="standardButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Standard Mode</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/QtWidgetsIcons.qrc">
       <normaloff>:/VatesSimpleGuiQtWidgets/icons/standard_view.png</normaloff>:/VatesSimpleGuiQtWidgets/icons/standard_view.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="multiSliceButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>MultiSlice Mode</string>
     </property>
     <property name="icon">
      <iconset resource="../../icons/QtWidgetsIcons.qrc">
       <normaloff>:/VatesSimpleGuiQtWidgets/icons/multislice_view.png</normaloff>:/VatesSimpleGuiQtWidgets/icons/multislice_view.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="threeSliceButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Three Slice Mode</string>
     </property>
     <property name="whatsThis">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../icons/QtWidgetsIcons.qrc">
       <normaloff>:/VatesSimpleGuiQtWidgets/icons/threeslice_view.png</normaloff>:/VatesSimpleGuiQtWidgets/icons/threeslice_view.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="splatterPlotButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="toolTip">
      <string>Scatter Plot Mode</string>
     </property>
     <property name="icon">
      <iconset resource="../../../../ParaviewPlugins/ParaViewFilters/SplatterPlot/SplatterPlot.qrc">
       <normaloff>:/Splatter.png</normaloff>:/Splatter.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../icons/QtWidgetsIcons.qrc"/>
  <include location="../../../../ParaviewPlugins/ParaViewFilters/SplatterPlot/SplatterPlot.qrc"/>
 </resources>
 <connections/>
</ui>
